app/Http/Controllers/Webapp/CategoryController.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace App\Http\Controllers\Webapp;
+
+use App\Http\Requests;
+use App\Libs\CdsApiDriver;
+use App\Models\Category;
+use App\Models\Post;
+use App\Models\Site;
+use Illuminate\Http\Request;
+
+class CategoryController extends NeedAuthController {
+
+  public function getListCategories(Request $request) {
+    $categories = Category::where('site_id', $request->get('id'))->get();
+
+    $listCategories = [];
+    foreach ($categories as $category) {
+      $tmpObj = new \stdClass();
+
+      $tmpObj->id = $category->id;
+      $tmpObj->parent = ($category->parent == 0 ? '#' : $category->parent);
+      $tmpObj->text = $category->name;
+
+      array_push($listCategories, $tmpObj);
+    }
+    
+    return response()->json($listCategories);
+  }
+
+  public function getCategorySync(Request $request) {
+    if (empty($site = Site::find($request->get('id')))) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "$site" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.relipa.blog/coding_rules#rule_key=php%3AS1481)